aten/src/ATen/native/native_functions.yaml

```diff
@@ -283,7 +283,7 @@
   use_c10_dispatcher: full
   variants: function
   dispatch:
-    DefaultBackend: view_as_real
+    CPU, CUDA: view_as_real
```

Comment:
I guess the same is true for view_as_complex just below?

Reply:
ah nice catch!